dss-api.yml

```diff
@@ -84,6 +84,9 @@ paths:
               results:
                 description: Results matching the `es_query`.
                 type: array
+              count:
```

Comment:
consider making this `total_hits` as that is considerably less ambiguous.

Reply:
sounds good